source/js/carbonads.js

```diff
@@ -0,0 +1,174 @@
+export default function() {
+  var _native = (function () {
```

Comment:
@mmollick This javascript was provided by carbonads and they call _native from the global scope here https://github.com/doctrine/doctrine-website/pull/271/files#diff-b94a2300afd9b8a8da845e9e7bc1a47cR201

What is the proper way to do this to make something available to be called?